docs/Makefile

```diff
@@ -0,0 +1,37 @@
+# Makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line.
+SPHINXBUILD   = sphinx-build
+SPHINXOPTS    =
+BUILDDIR      = build
+
+# Internal variables.
+SOURCEDIR      = .
+HTMLDIR        = $(BUILDDIR)/html
+SPELLINGDIR    = $(BUILDDIR)/spelling
+ALLSPHINXOPTS  = -d $(BUILDDIR)/doctrees $(SPHINXOPTS)
+
+.PHONY: help
+help:
+	@echo "Please use \`make <target>' where <target> is one of"
+	@echo "  help       Print this help"
+	@echo "  clean      Remove $(BUILDDIR) directory"
+	@echo "  html       Generate HTML documentation in $(HTMLDIR) directory"
+
+# @echo "  test       to check the spelling, and the source code"
```

Comment:
Why have this line commented when the `test` option is valid as we can see later in this same file?